CHANGELOG.MD

```diff
@@ -1,5 +1,24 @@
 # Mattermost Android Application Changelog
 
+## Release v3.3.0  
+
+Released 2016-08-16
+
+### Improvements
+- Enter key now creates a new line instead of sending the message.
+- Links added to the mobile apps in the welcome email, tutorial, and main menu.
```

Comment:
Wondering if we should summarize these two points into something easier to understand, or something that is more applicable to the mobile apps themselves. 

Like - "Made it easier for users to discover the mobile app by adding a download page that appears when the site is accessed on a mobile web browser. Also added download links to the welcome email, tutorial, and main menu."